docs/pages/admin-guides/management/diagnostics/metrics.mdx

```diff
@@ -23,6 +23,12 @@ bandwidth, and open file descriptors).
 This will enable the `http://127.0.0.1:3000/metrics` endpoint, which serves the
 metrics that Teleport tracks. It is compatible with [Prometheus](https://prometheus.io/) collectors.
 
+<Admonition type="tip" title="Grafana dashboard">
+
+Grafana dashboard template can be found at [examples/grafana/teleport-dashboard.json](https://github.com/gravitational/teleport/blob/branch/v(=teleport.major_version=)/examples/grafana/teleport-dashboard.json).
```

Comment:
The link 404s on the preview site. The path is `https://github.com/gravitational/teleport/blob/branch/v17/examples/grafana/teleport-dashboard.json`.

We could also use a pattern similar to the one we use for the avatar link in some of our Access Request plugin guides (e.g., [this one](https://github.com/gravitational/teleport/blob/09ff2c971aca2d8383f89985c12799065315f54c/docs/pages/identity-governance/access-request-plugins/ssh-approval-discord.mdx#L8)) to create a download link.